airflow/hooks/druid_hook.py

```diff
@@ -13,179 +13,71 @@
 # limitations under the License.
 
 from __future__ import print_function
-import logging
-import json
-import time
 
-from pydruid.client import PyDruid
+import logging
 import requests
+import time
 
-from airflow.hooks.base_hook import BaseHook
 from airflow.exceptions import AirflowException
-
-LOAD_CHECK_INTERVAL = 5
-DEFAULT_TARGET_PARTITION_SIZE = 5000000
-
-
-class AirflowDruidLoadException(AirflowException):
-    pass
+from airflow.hooks.base_hook import BaseHook
 
 
 class DruidHook(BaseHook):
-    '''
-    Interact with druid.
-    '''
-
-    def __init__(
-            self,
-            druid_query_conn_id='druid_query_default',
-            druid_ingest_conn_id='druid_ingest_default'):
-        self.druid_query_conn_id = druid_query_conn_id
-        self.druid_ingest_conn_id = druid_ingest_conn_id
+    """
+    Connection to Druid
+    """
+
+    def __init__(self,
+                 conn_id='druid_ingest_default',
+                 timeout=1,
+                 max_ingestion_time=18000):
+        self.conn_id = conn_id
+        self.timeout = timeout
+        self.max_ingestion_time = max_ingestion_time
         self.header = {'content-type': 'application/json'}
 
     def get_conn(self):
```

Comment:
The convention in hook is for `get_conn` to return an activated/configured client. Returning a URL deviates from that convention. Might has well implement `get_conn_url` instead and let `get_conn` raise a NotImplementedError